README.md

```diff
@@ -59,18 +59,22 @@ pip install pyro-ppl[extras]  # for running examples/tutorials
 
 For recent features you can install Pyro from source.
 
-First install a recent PyTorch, currently PyTorch commit `e40425f`.
 ```sh
-git clone git@github.com:pytorch/pytorch
+git clone --recursive https://github.com/pytorch/pytorch
 cd pytorch
-git checkout [commit hash above]
+git checkout e40425f  # <---- a well-tested commit
 ```
 Then build PyTorch following instructions in the PyTorch
 [README](https://github.com/pytorch/pytorch/blob/master/README.md).
+Depending on your system, this may be as simple as
+```sh
+pip install ninja
+python setup.py install
```

Comment:
for me the install build fails if i have `ninja`, i had to uninstall `ninja` for build install (vs build develop)

Reply:
@jpchen can you point me to your PR that fixes this by removing this line?